lib/src/components/Fields/Section.tsx

```diff
@@ -12,19 +15,35 @@ const messages = defineMessages({
   },
 });
 
+/**
+ * Manage conditions to display or not the dropzone at the top of a section
+ * It must not display a dropzone in a section that contains other sections to
+ * avoid using widgets and sections at the same level
+ *
+ * @param id of the section
+ * @param schema is the whole schema loaded in the library
+ * @param selector is the section name if one is selected
+ */
+function doesContainSection(id: string, schema: VernaSchemaType, selector: Maybe<string>) {
+  if (selector || id !== 'root') return false;
+
+  let containObject = false;
+  _.forEach(schema.formSchema?.properties, (section) => {
+    if (section.type === 'object') containObject = true;
+  });
+
+  return containObject;
```

Comment:
Ah i was thinking it was right while reading it, but the tests revealed the truth ! It doesn't work because the return true is in the forEach so it return in the forEach not the whole function :)
But if you know a lodash function that could return at the first occurrence to avoid parsing the whole object i would be interested in that but i couldn't find it.